src/panel1.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html id="1">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <title>Facebook Container</title>
+    <link rel="stylesheet" href="panel.css">
+    <script src="background.js"></script>
```

Comment:
The background script should not be included in the popup (same for `panel2.html`) - that's also why some tests are failing (code is executed twice).